pydcs_extensions/fa18efg/fa18efg.py

```diff
@@ -153,13 +252,16 @@ class Pylon1:
         # ERRR {CATM-9M}
         # ERRR {CATM-9X}
         AIM_9X_Sidewinder_IR_AAM = (1, Weapons.AIM_9X_Sidewinder_IR_AAM)
-        AN_ASQ_T50_TCTS_Pod___ACMI_Pod = (1, Weapons.AN_ASQ_T50_TCTS_Pod___ACMI_Pod)
+        AN_ASQ_T50_TCTS_Pod___ACMI_Pod = (
+            1,
+            Weapons.AN_ASQ_T50_TCTS_Pod___ACMI_Pod,
+        )
 
     class Pylon2:
         TLAU_127 = (2, WeaponsFA18EFG.TLAU_127)
-        AIM_120B_AMRAAM___Active_Radar_AAM = (
+        AIM_120D_AMRAAM___Active_Rdr_AAM = (
```

Comment:
Are you sure we need to replace the B with the D model? Perhaps we need to add them both?

The main reason I think adding both would make sense is an attempt at backwards-compatibility with earlier versions of the superbug, because iirc the B model would be used as a D model, but I believe @MetalStormGhost can shed more light on this 😅

Reply:
I consulted with Ghosti and we believe this change should also be reverted. The CLSIDs for the B & D models are the same, and so this would "overwrite" the B model altogether.

I believe that's why we left the D model out, but choosing a B model will eventually result in a D model in-game if I understood correctly.

Reply:
I've removed the AIM-120D in the last push